modules/network/networker.go

```diff
@@ -164,13 +255,43 @@ func (n *networker) ApplyNetResource(network modules.Network) (string, error) {
 
 	// map the network ID to the network namespace
 	path := filepath.Join(n.storageDir, string(network.NetID))
-	if err := ioutil.WriteFile(path, []byte(nibble.NetworkName()), 0660); err != nil {
+	file, err := os.Create(path)
+	if err != nil {
 		return "", err
 	}
+	defer file.Close()
+
+	enc := json.NewEncoder(file)
+	if err := enc.Encode(network); err != nil {
```

Comment:
I know store the full network object in the network file. I thought may be we should only store the local NR for this network, but it seems i need more info to create a nibble for that network, that is associated with the full network object.